homeassistant/components/tuya/const.py

```diff
@@ -187,6 +187,9 @@ class DPCode(str, Enum):
     FAN_SPEED_PERCENT = "fan_speed_percent"  # Stepless speed
     FAR_DETECTION = "far_detection"
     FAULT = "fault"
+    FEED_MANUAL = "manual_feed"
```

Comment:
```suggestion
    MANUAL_FEED = "manual_feed"
```

Reply:
I was 99% sure you will point this out ;)